client/spago.dhall

```diff
@@ -38,8 +46,9 @@
   , "transformers"
   , "tuples"
   , "unfoldable"
+  , "unsafe-coerce"
   , "web-html"
   ]
 , packages = ./packages.dhall
-, sources = [ "src/**/*.purs", "test/**/*.purs" ]
+, sources = [ "src/**/*.purs", "config/dev/**/*.purs", "test/**/*.purs" ]
```

Comment:
Perhaps we should use 3 `spago.dhall` files: a shared one, one for dev and one for prod?